### ISSUE TITLE
Is DeSo using web3 infrastructure for medias?

### ISSUE THREAD

**Issue by b4s36t4** [deso-protocol/backend#385]
Hey all.

I have seen that media.go is using gcp as the media provider, is that so? Why can't you use web3 infrastructure like stroj or filecoin or arcana?

Does using GCP makes deso fully web3 compliance?

**Comment by ItsAditya-xyz** [comment]
no, all media is stored on centralized servers. although, you can spin up IPFS/filecoin/arweave to have it have the media on decentralized infra
